### PR TITLE
Contribute DSTIM workbook

### PR DESCRIPTION
**Pull request by avital-m**
Fixes #

## Proposed Changes

  -Contribute a new workbook to sentinel for DSTIM.

### CODE REVIEW

**Comment by v-sabiraj** [comment]
Hey @avital-m , We have resolved the issue for PR Validations failure and request you to please update your PR branch with the latest changes from **Azure:master** so that the PR Validations can be executed.... Thanks!!!

**Comment by avital-m** [comment]
> 

Hi @v-sabiraj I updated the pr. Pease let me know if any other modifications are needed.

File: Sample Data/Custom/DSTIMAccess_CL.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
```
Comment (v-jayakal) [inline Sample Data/Custom/DSTIMAccess_CL.json:1]: Should be an array, braces [ and ] are missing.

File: Sample Data/Custom/DSTIMAccess_CL.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	"TenantId": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+	"SourceSystem": "RestAPI",
+	"MG": "",
+	"ManagementGroupName": "",
+	"TimeGenerated": "2021-11-18T21:58:38.0289591Z",
+	"Computer": "",
+	"RawData": "",
+	"TimeDiscovered_t": "2021-06-18T21:58:38.0289591Z",
```
Comment (v-jayakal) [inline Sample Data/Custom/DSTIMAccess_CL.json:9]: can you please remove the suffix like _s, _t etc, they will be added automatically when the data is ingested.

**Review by v-jayakal — CHANGES_REQUESTED** [review]

**Comment by v-jayakal** [comment]
@avital-m - There are conflicts in workbookmetadata file, please resolve, get latest from master and merge to resolve. Thanks

File: Sample Data/Custom/DSTIMFunctions/DSTIMCorrelatedLogs.kql [hunk 3]
```diff
@@ -0,0 +1,41 @@
+let DSTIMCorrelatedLogs = (_startDate:datetime, _endDate: datetime, _subscriptions:dynamic = dynamic("*"), _resourceGroups:dynamic = dynamic(["*"]), _userAccounts:dynamic = dynamic(["*"]))
```
[the remark this answers is not in the document]
Comment (avital-m) [reply Sample Data/Custom/DSTIMFunctions/DSTIMCorrelatedLogs.kql:1]: @v-jayakal how should I define it if my function gets parameters?  and also, will this function be available for users as part of the template?

File: Sample Data/Custom/DSTIMAccess_CL.json (hunk 1 above)
Comment (avital-m) [reply Sample Data/Custom/DSTIMAccess_CL.json]: @v-jayakal fixed

File: Sample Data/Custom/DSTIMAccess_CL.json (hunk 2 above)
Comment (avital-m) [reply Sample Data/Custom/DSTIMAccess_CL.json]: @v-jayakal fixed

**Comment by v-sabiraj** [comment]
The preview image file names must include "Black" and "White" instead of "Dark" and "Light".

**Comment by avital-m** [comment]
> The preview image file names must include "Black" and "White" instead of "Dark" and "Light".

@v-jayakal / @v-sabiraj I updated the images

File: Sample Data/Custom/DSTIMAccess_CL.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    "OperationName": "GetBlob",
+    "StatusCode": "200",
+    "Uri": "https://aipclptest.blob.core.windows.net/test2/es-ssn (1).docx",
+    "CallerIPAddress": "10.1.0.38",
+    "UserAgentHeader": "Azure-Storage/9.3.0 (.NET CLR 4.0.30319.42000; Win32NT 6.2.9200.0)",
+    "ResourceSubscriptionId": "bbbabe37-eda3-48ea-98ed-f0a70c31a45b",
+    "ResourceGroup": "BreachManagement-dev",
+    "StorageAccountName": "breachmanagement-dev",
+    "ResponseBodySize": "26070",
+    "RequesterUpn": "MeganB@seccxp.ninja",
```
Comment (NikTripathi) [inline Sample Data/Custom/DSTIMAccess_CL.json:21]: Email should be sanitized@sanitized.com.

**Review by NikTripathi — CHANGES_REQUESTED** [review]

File: Sample Data/Custom/DSTIMAccess_CL.json (hunk 4 above)
Comment (avital-m) [reply Sample Data/Custom/DSTIMAccess_CL.json]: @NikTripathi fixed.

File: Workbooks/DSTIMWorkbook.json [hunk 5]
```diff
@@ -0,0 +1,2286 @@
+{
```
Comment (v-sumdh) [inline Workbooks/DSTIMWorkbook.json]: It seems DSTIMWorkbook.json format are not as per other workbooks json format. Please refer any other workbooks format  https://github.com/Azure/Azure-Sentinel/tree/master/Workbooks .Thanks

File: Parsers/DSTIM/GetClassificationList.txt [hunk 6]
```diff
@@ -0,0 +1,12 @@
+// Usage Instruction : 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias as GetClassificationList.
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. GetClassificationList | take 10).
+// Reference : Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+let GetClassificationList = (_startDate:datetime, _endDate: datetime, _subscriptions:dynamic = dynamic("*"), _resourceGroups:dynamic = dynamic(["*"]), _userAccounts:dynamic = dynamic(["*"])) 
+{ 
+    DSTIMCorrelatedLogs(_dateStart, _dateEnd, _subscriptions, _resourceGroups, _userAccounts)
```
Comment (v-sumdh) [inline Parsers/DSTIM/GetClassificationList.txt:7]: It seems parameter names like _startDate,_endDate should be same in function.Thanks

Comment (avital-m) [reply Parsers/DSTIM/GetClassificationList.txt]: @v-sumdh fixed.

**Pull request merged**